lib/internal/Magento/Framework/ObjectManager/ObjectManager.php

```diff
@@ -53,7 +53,6 @@ public function __construct(FactoryInterface $factory, ConfigInterface $config,
      */
     public function create($type, array $arguments = [])
     {
-        $type = ltrim($type, '\\');
         return $this->_factory->create($this->_config->getPreference($type), $arguments);
```

Comment:
@ishakhsuvarov can i remove these changes and sent only changes related to this PR?